src/main/java/sg/ncl/MainController.java

```diff
@@ -95,6 +95,7 @@
 
     private static final String TEAM_NAME = "teamName";
     private static final String NODE_ID = "nodeId";
+    private static final String PERMISSION_DENIED = "Permission denied";
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "Permission denied" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)